sessions/sessions.go

```diff
@@ -180,6 +179,31 @@ func (s *Sessions) GetSession(region, instance string) (*session.Session, *Insta
 	return nil, nil
 }
 
+func buildCredentials(instance config.Instance) (*credentials.Credentials, error) {
+	if instance.AWSRoleARN != "" {
+		os.Setenv("AWS_ACCESS_KEY_ID", instance.AWSAccessKey)
```

Comment:
Hi,
Instead of setting env vars can we do something like: 
```
    awsSession = session.Must(session.NewSession(&aws.Config{
        Region:      aws.String("ap-southeast-1"),
        Credentials: credentials.NewStaticCredentials("YOUR_ACCESS_KEY","YOUR SECRET_KEY", ""),
    }))
```
I would prefer that but if it is not possible (I haven't tested my suggestion) at least we must clean the env vars once we have the session.

Reply:
@percona-csalguero Took care of this